tests/Unit/TestCreation.hpp

```diff
@@ -48,10 +48,12 @@ struct TestCreationOpt {
 /// \ingroup TestingFrameworkGroup
 /// Construct an object from a given string.
 ///
-/// The string must not contain the
-/// name of the option (which is the name of the struct by default if no
-/// `name()` function is present). For example, to create a struct with a
-/// `SizeT` named option the following would be used:
+/// The string must not contain the name of the option (specifically, the class
```

Comment:
Should "class" -> "struct" for consistency with use of "struct" later in this sentence?